modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -721,7 +737,7 @@ private void _addObjectDefinitionResources(
 			ObjectAction[] objectActions, long objectDefinitionId,
 			ObjectLayout[] objectLayouts,
 			ObjectRelationship[] objectRelationships,
-			ObjectValidationRule[] objectValidationRules,
+			List<ObjectValidationRule> objectValidationRules,
```

Comment:
Instead of changing the parameter type here, you can pass `objectValidationRules.toArray(new ObjectValidationRule[0])` just in the put method and leave the rest as it is.